Userland/Libraries/LibC/bits/stdint.h

```diff
@@ -138,4 +138,45 @@ typedef __INTMAX_TYPE__ intmax_t;
 #define PTRDIFF_MAX __PTRDIFF_MAX__
 #define PTRDIFF_MIN (-__PTRDIFF_MAX__ - 1)
 
+#define INT8_WIDTH 8
```

Comment:
It might come from a difference of revision of the standard, but I can't find any `INTN_WIDTH` in the 7.20.2 (Limits of specified-width integer types) paragraph.

Reply:
It's on the latest draft for C23
https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2731.pdf

Reply:
Ok, lgtm then